test/guest/cockpit.conf

```diff
@@ -1,7 +1,7 @@
-{ "tags": { "fedora-22":         "21",
-            "rhel-7":            "12",
-            "fedora-atomic-22":  "4",
-            "fedora-rawhide":    "7",
-            "fedora-22-testing": "6"
+{ "tags": { "fedora-22":         "22",
```

Comment:
The new images break testBond test. With the old images, that test works. Not sure why that is.